chp02_forces/NOC_2_05_fluidresistance/sketch.js

```diff
@@ -29,7 +27,9 @@ function setup() {
 }
 
 function draw() {
-  background(127);
+  background(255);
+  fill(0);
+  text("click to reset",10,100);
```

Comment:
Hi @jefarrell, I was actually using p5.dom for instructions to appear below canvas as `<p>` elements.  Was that not working for you?